Library/Formula/splint.rb

```diff
@@ -30,7 +30,7 @@ def install
     EOS
 
     output = `#{bin}/splint #{path} 2>&1`
-    assert output.include?("5:18: Variable c used before definition")
+    assert "5:18: Variable c used before definition", output
     assert_equal 1, $?.exitstatus
```

Comment:
You can remove this line if you use `shell_output` above with `1` as the second parameter?